src/components/MyRequestsPage.jsx

```diff
@@ -1,34 +1,77 @@
-import React from "react";
+import React, { useState } from "react";
 import MyListComponent from "./MyListComponent";
-import { Menu, Button } from "semantic-ui-react";
+import { Button } from "semantic-ui-react";
 import { useSelector } from "react-redux";
 import Offers from "./Offers";
 import { Link, Redirect } from "react-router-dom";
 
 const MyRequestsPage = () => {
-  const mySelectedRequest = useSelector((state) => state.requests.mySelectedRequest);
+  const mySelectedRequest = useSelector(
+    (state) => state.requests.mySelectedRequest
+  );
+  const [selectedStatus, setSelectedStatus] = useState("pending");
   const authenticated = useSelector(
     (state) => state.authentication.authenticated
   );
 
+  const showMyRequests = (status) => {
+    setSelectedStatus(status);
+  };
+
+  const activeMenuItem = (menuItem) => {
```

Comment:
It would be better to do this with a className instead, and put css in the css file.

Reply:
Maybe just do className={ selectedStatus === "pending" && "active"} could do the trick, instead of separate function for it.

Reply:
I believe changes are not here anymore